### ISSUE TITLE
Feature Request: Add parameter options just like priority: like label and project

### ISSUE THREAD

**Issue by 0xTripbee** [hkgnp/logseq-todoist-plugin#12]
It would be HUGEEE!! For my workflow I have already some automation and some processes ready and waiting for those features ^^

Best regards,
AL

**Comment by hkgnp** [comment]
Hi! Do you mocking up a block of what it should look like?

Not sure if I understand you correctly.

**Comment by PyPlot** [comment]
Sure, so just like (atm) we can send the priority argument like this
![Screenshot 2021-12-18 at 16 32 14](https://user-images.githubusercontent.com/49495734/146648626-8cda0fa6-d751-4c0a-a19e-c45e05dcc131.png)

I'm suggesting a feature improvement that could take 2 more arguments: project and label (both IDs).
And maybe the labels argument could take a list/array where the user separates (the label IDs) with commas

![Screenshot 2021-12-18 at 16 36 00](https://user-images.githubusercontent.com/49495734/146648700-0ec9f22f-ce88-410b-ab75-199f68233f46.png)
![Screenshot 2021-12-18 at 16 36 08](https://user-images.githubusercontent.com/49495734/146648703-4299eda3-1bb9-45d8-b596-c5355f540b24.png)

**Comment by hkgnp** [comment]
Hey @PyPlot, sorry for the late reply! While getting the Project ID for this feature request is straightforward, may I know how will you get the label IDs?

Based on what I know, the only way to get them is to actually test the API (e.g. curl, Python).

**Comment by PyPlot** [comment]
Ohh @hkgnp I think I didn't express myself my goal.

My feature request is more lean to the /send command, so we can specify a project for the task to go to (be attached to) and to be able to specify also labels when sending the task to Todoist. (and because projects and labels are referenced by ID's that's why I mentioned them).

TLDR: My objective is to (just like priority) to be able to set a Project and Labels when creating a task on Logseq to send to Todoist

**Comment by hkgnp** [comment]
I see! Okay it's a bit clearer. 

But can I check with you how you are able to get the IDs for the labels? They are not available on Todoist.com or their app.

**Comment by PyPlot** [comment]
Yeah I don't have a solution for that because I never did programming with Todoist, but I can try to do some research. I'm only aware of its existence because in my integrations (like with Automate.io for example), one can extract/send Label IDs. Oh and other thing, there is an macOS Alfred Workflow for Todoist that allows a user to specify the Project and Label, but by name directly (instead of having to get the ID).

Maybe instead by name should be not only more user-friendly and easier to use but also easier to make

**Comment by hkgnp** [comment]
Ah, this is much much clearer! I know what you mean now.

I have an idea but will need some time because work is quite busy! Will let you know once there is an update!

**Comment by PyPlot** [comment]
Thank you so much for the quick response and follow up!! :)

Ofc, no rush. I'm going through the same...crazy times we're living on...

Just let me know if you need me to do anything to work that with you 👍

**Comment by hkgnp** [comment]
Hi, I may need more time to implement your FR, but I've did a change to the current UI where you can specify the tasks be sent to a specific project or task. However, this will apply to all the tasks that you send.

<img width="392" alt="Screenshot 2021-12-22 at 12 42 59 PM" src="https://user-images.githubusercontent.com/47936793/147041756-d25abef3-f5e1-442f-b452-01ffdc273293.png">

A workaround is to change the settings each time you want to do it, but understand that this is not the most ideal if you are sending tasks to different projects all the time. The tricky thing with specifying the project and label by typing them out in the block instead of via a dropdown is that the chance of typos are there.

**Comment by PyPlot** [comment]
Awesome, yeah that would be temp workaround for sure!

Yeah and I understand, the typo error prone gets introduced there, but still easier then referring by ID (and better than don't be able to do it at all)... at least in my point of view. Ofc someone else may experience or look at it completely differently :)

**Comment by hkgnp** [comment]
Breaking change: Did a major refactor. Now when you send tasks, there is a popup to indicate the above parameters. Do update if you require the change.

**Comment by b0lle** [comment]
Hey @hkgnp,

i am on version 5.2.1 and tried your latest "breaking change". At least for me there is no pop-up window if I send a task to todoist.
Even if I configure some properties like
```
This is a task
priority:: 1
project:: myproject
```
those properties wont pass to todoist (mac v3229). Is there any setting I missed?

**Comment by hkgnp** [comment]
Available in latest version.
